source/guides/clear/performance.rst

```diff
@@ -0,0 +1,236 @@
+		the build system may still need patching. A manual build will not ignore
+		the build system override values if they exist.
+
+Global flags
+	Compiler flags applied at a global level for all packages. The |CL| RPM
+	configuration (`clr-rpm-config <https://github.com/clearlinux/clr-rpm-config>`_)
+	contains global compiler flags. Search the :file:`macros` file for
+	:command:`global_cflags` and search the :file:`rpmrc` file for
+	:command:`optflags`. Global compiler flags may be overridden.
+
+	.. note::
+		|CL| doesn't use RPMs to install software. |CL|
+		distributes software in the form of :ref:`bundles-guide`. The RPM format
+		is only used during the |CL| build process as a way to resolve
+		dependencies.
+
+Per-package flags
+	Compiler flags applied at a per-package level. The package's autospec
+	repository contains the package-specific compiler flags. Search the
+	:file:`.spec` file for the
+	section starting with :command:`export CFLAGS`.
+
+Multiple builds of libraries with CPU-specific optimizations
+============================================================
+
+To fully use the capabilities in different generations of CPU hardware, |CL|
+will perform multiple builds of libraries with CPU-specific optimizations. For
+example, |CL| builds libraries with Intel® Advanced Vector Extensions 2 (Intel®
+AVX2) and Intel® Advanced Vector Extensions 512 (Intel® AVX-512). |CL| can then
+dynamically link to the library with the newest optimization based on the
+processor in the running system. Runtime libraries used by ordinary applications
+benefit from these CPU specific optimizations.
+
+The autospec repository for Python* shows an example of this optimization:
+https://github.com/clearlinux-pkgs/python3
+
+Kernel
+******
+
+A modern kernel with variants optimized for different platforms
+===============================================================
+
+|CL| is a rolling release distribution that uses the newest upstream Linux
+kernel. The Linux kernel has frequent updates which can include performance
+enhancements. It's a policy of the |CL| team to try to upstream any performance
+enhancements in the Linux kernel for all to use.
+
+|CL| `builds different kernel variants <https://docs.01.org/clearlinux/latest/guides/clear/compatible-kernels.html>`_ for compatibility with specific platforms.
+For example, kernels meant to run on virtual machines skip support for much of
+the physical hardware that doesn’t show up in VM environments and will slow down
+boot.
+
+View the kernel configuration and patches to the default native kernel in the
+autospec repository: https://github.com/clearlinux-pkgs/linux/
+
+Utility to enforce kernel runtime parameters
+============================================
+
+The Linux kernel exposes parameters for tuning the behavior of drivers and
+devices such as certain buffers and resource management strategies. |CL| uses a
+small utility, `lr-power-tweaks <https://github.com/clearlinux-pkgs/clr-power-tweaks>`_,

```

Comment:
Ln 195: Typo: Missing "c"

Change to: 
`clr-power-tweaks....`

Reply:
good catch!